### PR TITLE
Fix imports from iiblib [CLOUDDST-3064]

### PR DESCRIPTION
**Pull request by pbortlov**
* fix iiblib imports
* remove unused imports

### CODE REVIEW

**Comment by midnightercz** [comment]
Looks good but needs to be rebased on https://github.com/release-engineering/iiblib/pull/25

**Comment by pbortlov** [comment]
> Looks good but needs to be rebased on [release-engineering/iiblib#25](https://github.com/release-engineering/iiblib/pull/25)

Yeah, we will wait when is the iiblib merged and then we will rebase.

**Comment by lipoja** [comment]
LGTM - once the required PR is merged I will approve it.

**Review by lipoja — APPROVED** [review]

**Review by midnightercz — APPROVED** [review]

**Pull request merged**
